pkg/vendors/mellanox/mellanox.go

```diff
@@ -141,6 +142,20 @@ func (m *mellanoxHelper) GetMellanoxBlueFieldMode(PciAddress string) (BlueFieldM
 	return -1, fmt.Errorf("MellanoxBlueFieldMode(): unknown device status for %s", PciAddress)
 }
 
+func (m *mellanoxHelper) MlxResetFW(pciAddresses []string) error {
+	log.Log.Info("mellanox-plugin resetFW()")
+	for _, pciAddress := range pciAddresses {
+		cmdArgs := []string{"-d", pciAddress, "-l", "3", "-y", "reset"}
```

Comment:
@noama-nv at this point the node is drained from workloads. id prefer to not send reset signal while driver is running.
if we find its problematic we can switch. either way node will reboot after.

during @tobiasgiese testing we didnt encounter issues with the current way of doing reset.

Reply:
@tobiasgiese i hade another thought about this. since we are going to reboot right after, lets skip the driver restart (adding --skip_driver) as it will save us much time (especially on systems with multiple NICs).